paddle/framework/framework.proto

```diff
@@ -111,6 +111,7 @@ message VarDesc {
   enum VarType {
     LOD_TENSOR = 1;
     SELECTED_ROWS = 2;
+    FEED_FETCH_LIST = 3;
```

Comment:
Should separate FeedOp's input type and FetchOp's output type?

- MINIBATCH, and
- FETCHED